mappings/net/minecraft/world/World.mapping

```diff
@@ -175,8 +178,8 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 	METHOD method_8508 updateNeighborsExcept (Lnet/minecraft/class_2338;Lnet/minecraft/class_2248;Lnet/minecraft/class_2350;)V
 		ARG 1 pos
 		ARG 2 sourceBlock
-	METHOD method_8509 setTicksSinceLightning (I)V
-		ARG 1 ticksSinceLightning
+	METHOD method_8509 setLightningBrightTicks (I)V
+		ARG 1 lightningBrightTicks
```

Comment:
the ticks the world should be rendered bright due to a lightning. on client a lightning sets it to 2. decreases 1 per tick.

Reply:
So this is decreased every tick? If yes, I think `getLightningTicksRemaining` would be better. Otherwise, someone might think it's a constant value for how long all lightnings are.